src/app/shared/cookies/orejime-configuration.ts

```diff
@@ -0,0 +1,182 @@
+import {
+  IMPERSONATING_COOKIE,
+  REDIRECT_COOKIE,
+} from '../../core/auth/auth.service';
+import { TOKENITEM } from '../../core/auth/models/auth-token-info.model';
+import {
+  CAPTCHA_COOKIE,
+  CAPTCHA_NAME,
+} from '../../core/google-recaptcha/google-recaptcha.service';
+import { LANG_COOKIE } from '../../core/locale/locale.service';
+import { NativeWindowRef } from '../../core/services/window.service';
+
+/**
+ * Cookie for has_agreed_end_user
+ */
+export const HAS_AGREED_END_USER = 'dsHasAgreedEndUser';
+
+export const ANONYMOUS_STORAGE_NAME_OREJIME = 'orejime-anonymous';
```

Comment:
Same here, I think this is called `orejime`.   I've not yet seen a cookie named `orijime-anonymous`.